common/src/main/java/com/redhat/cloud/notifications/models/DrawerNotificationId.java

```diff
@@ -0,0 +1,53 @@
+package com.redhat.cloud.notifications.models;
+
+import jakarta.persistence.Embeddable;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.io.Serializable;
+
+
+@Embeddable
+public class DrawerNotificationId implements Serializable {
+
+    @NotNull
+    @Size(max = 50)
+    private String orgId;
+
+    @NotNull
+    @Size(max = 50)
+    public String userId;
```

Comment:
The naming 'userId' is already used on 'email_subscription' table, this is why I kept the same wording.
In addition, because username is also a Postgres global variable, it's a little bit more tricky to use on a stored fuction :)

Reply:
😢